TensorLib/NonEmptyList.lean

```diff
@@ -0,0 +1,25 @@
+namespace TensorLib
+
```

Comment:
Use a section?

Reply:
I don't grok the rules yet. I still need to type `NonEmptyList a` everywhere even with a section defining `variable (a: Type)`. So nothing was gained when I tried this. What am I missing?

Reply:
K, did this and things are slowly getting better